addons/stock/views/product_views.xml

```diff
@@ -511,6 +511,9 @@
             <field name="priority" eval="100"/>
             <field name="arch" type="xml">
                 <tree sample="1" js_class="stock_report_list_view">
+                    <header>
+                        <button name="action_inventory_at_date" invisible="((context.get('inventory_mode') and not context.get('inventory_report_mode')) or context.get('no_at_date'))" string="Inventory at Date" type="object" class="btn-primary ms-1" display="always"/>
```

Comment:
I see the occurrence in 2 other views. Clean them too. Remove the action in python code of quant